frontend/src/components/ui/Navbar/Navbar.tsx

```diff
@@ -1,12 +1,23 @@
+'use client';
+
 import Link from 'next/link';
 import styles from './Navbar.module.css';
+import { useAuth } from '@/context/auth/auth';
+import { configuration } from '@/config/Constants';
+
+const BACKEND_URL = configuration.url.BACKEND_API_URL;
+
+const Navbar = () => {
+	const { isAuthenticated } = useAuth();
```

Comment:
Should this be checked in the ¿navbar?

Reply:
I thought it would be convenient to have a Log In/Log out in the navbar but if that's not what we want I can revise 😄

Reply:
I misread this; sorry!